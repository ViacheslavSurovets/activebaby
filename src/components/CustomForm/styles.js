import styled from 'styled-components';

export const Form = styled.form`
  display: flex;
  flex-direction: column;
  justify-content: center;
  margin-top: ${ props => props.marginTop }rem;
  margin-bottom: ${ props => props.marginBottom }rem;
  margin-right: ${ props => props.marginRight }rem;
  margin-left: ${ props => props.marginLeft }rem;
`;
