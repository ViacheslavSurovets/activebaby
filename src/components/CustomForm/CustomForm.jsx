import React from 'react';
import { Form } from './styles';
import PropTypes from 'prop-types';

const CustomForm = ( { children, ...rest } ) => (
  <Form { ...rest }>
    { children }
  </Form>
);

export default CustomForm;

CustomForm.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.array,
    PropTypes.object
  ])
};
