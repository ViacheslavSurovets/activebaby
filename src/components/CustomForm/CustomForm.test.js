import React from 'react';
import { shallow } from 'enzyme';
import CustomForm from './CustomForm';


describe ( 'CustomForm', () => {
  it ( 'CustomForm Snapshot', () => {
    expect ( shallow ( <CustomForm /> ) ).toMatchSnapshot ();
  } );
} );
