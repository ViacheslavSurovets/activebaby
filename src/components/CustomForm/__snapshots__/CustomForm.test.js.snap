// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomForm CustomForm Snapshot 1`] = `ShallowWrapper {}`;
